app/styles/components/events/view/export.scss

```diff
@@ -32,3 +32,11 @@
 .url {
   padding: 0 1em !important;
 }
+
+.copy-url {
+  width: 100%;
+}
+
+.display-url {
```

Comment:
Remove this class. On mobile view it causes this:
![image](https://user-images.githubusercontent.com/17252805/54871029-44e3a880-4de9-11e9-81c3-f55a232076f0.png)